.vscode/launch.json

```diff
@@ -152,7 +152,7 @@
                 "run",
                 "--headless",
                 "--browser",
-                "chrome"
+                "chrome",
```

Comment:
Trailing commas are not a part of JSON spec
Apparently vscode supports them, however it would be more correct to remove them, rather then add.